_data/users.yml

```diff
@@ -228,6 +228,16 @@
     contributed the majority of the Trino opensource codebase.
   urltext: Learn more about the Starburst products!
   url: https://www.starburst.io/
+- name: Stripe
+  anchor: stripe
+  logo: /assets/images/logos/stripe-blue.png
+  logosmall: /assets/images/logos/stripe-small.png
+  testimonial: |
+    As of September 3rd 2020, Sigma supports queries using Prestosql v334 SQL
+    syntax. As a part of this change, object (such as map, array, etc) values
+    are now encoded in JSON format.
+  urltext:
+  url: https://stripe.com/
```

Comment:
Should I link to the Stripe support site where they have a continued explanation: https://support.stripe.com/questions/why-has-the-format-of-object-value-in-my-sigma-query-result-changed